instrumentation/httpurlconnection/library/src/main/java/io/opentelemetry/instrumentation/library/httpurlconnection/HttpUrlReplacements.java

```diff
@@ -290,7 +290,9 @@ private static void endTracing(URLConnection connection, int responseCode, Throw
 
     private static void startTracingAtFirstConnection(URLConnection connection) {
         Context parentContext = Context.current();
-        if (!HttpUrlConnectionSingletons.instrumenter().shouldStart(parentContext, connection)) {
+        if (HttpUrlConnectionSingletons.instrumenter() == null
```

Comment:
I'd assign the `HttpUrlConnectionSingletons.instrumenter()` to a var and reuse it